docs/src/design/chiplets/hasher.md

```diff
@@ -363,7 +363,7 @@ $$
 
 In the above:
 
-- $m$ is a _transition label_ which uniquely identifies each transition function.
+- $m$ is a _transition label_, composed of the [operation label](main.md#operation-labels) and the periodic columns, which uniquely identify each transition function.
```

Comment:
It might be good to add one or two sentences here explaining why we mix in periodic columns into the label.

Reply:
I've added a bit more explanation. Please let me know what you think